GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -145,6 +146,64 @@ const float* Mesh::getBoundingBoxInfo() {
     return bounding_box_info_;
 }
 
+const float* Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat) {
```

Comment:
This method returns a `float*` ... which always points to a member field. Far better to take/modify a `float*` parameter.